services/cnv-qe/admin_rbac.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/description: Test infrastructure that supports cnv-qe
+    openshift.io/display-name: cnv-qe CI
+  name: cnv-qe
+---
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: image-puller
+  namespace: cnv-qe
+---
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: cnv-qe-image-puller-binding
+  namespace: ocp
+roleRef:
+  kind: ClusterRole
+  apiGroup: rbac.authorization.k8s.io
+  name: system:image-puller
+subjects:
+- kind: ServiceAccount
+  namespace: ocp
```

Comment:
I think you want that to point to `cnv-qe`

Reply:
yes, missed one. thank you.